jujugui/static/gui/src/app/components/deployment-flow/credential/add/add.js

```diff
@@ -231,8 +231,9 @@ YUI.add('deployment-credential-add', function() {
                 name="general-action-blue"
                 size="16" />
               Credentials are stored securely on our servers and we will
-              notify you by email whenever they are used. See where they are
-              used and manage or remove them via the account page.
+              notify you by email whenever they are changed or deleted. See
```

Comment:
Should this be "You can see where they are used and manage or remove them..." rather than "See ..."; the latter sounds a little awkward to me.